Explorer/Assets/DCL/Minimap/MinimapController.cs

```diff
@@ -221,7 +221,7 @@ private void SetWorldMode(bool isWorldModeActivated)
         public override void Dispose()
         {
             cts.SafeCancelAndDispose();
-            realmNavigator.OnRealmChanged -= OnRealmChanged;
+            realmNavigator.RealmChangedToGenesis -= OnRealmChangedToGenesis;

```

Comment:
I dont know if this name change is ideal. This action is called on any realm change, not only on realm change to genesis. And we have a boolean that indicates that (if its genesis or not).